submissions/andysmokk/classic-frogger-game/app.js

```diff
@@ -0,0 +1,117 @@
+// Enemies our player must avoid
+const INITIAL_POSITION_PLAYER_X = 203;
+const INITIAL_POSITION_ENEMIES_X = -200;
+const INITIAL_POSITION_ENEMY_ONE_Y = 62;
+const INITIAL_POSITION_ENEMY_TWO_Y = 145;
+const INITIAL_POSITION_ENEMY_THREE_Y = 228;
+const INITIAL_POSITION_PLAYER_Y = 390;
```

Comment:
Think of less wordy contants definition.
Requirements provide relevant examples.

Reply:
Classes do not refer to any global variables, like global variable player, which is an instance of Player class (referring to global constants and globals provided by the gaming platform like Resources is OK); Hint: pass instance of a game object (or objects) as an argument to other game objects they need to interact with.